checker_c#/N0t3b00kChecker/Db/CheckerDb.cs

```diff
@@ -0,0 +1,53 @@
+﻿using EnoCore.Checker;
+using Microsoft.Extensions.Logging;
+using MongoDB.Driver;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace N0t3b00kChecker.Db
+{
+    public class CheckerDb
+    {
+        private readonly ILogger<CheckerDb> logger;
+        private readonly IMongoCollection<NotebookUser> users;
+        private readonly InsertOneOptions insertOneOptions = new() { BypassDocumentValidation = false };
+
+        public CheckerDb(ILogger<CheckerDb> logger)
+        {
+            this.logger = logger;
+            this.logger.LogDebug($"CheckerDb() via {MongoConnection})");
+            var mongo = new MongoClient(MongoConnection);
+            var db = mongo.GetDatabase("N0t3b00kCheckerDatabase");
+            this.users = db.GetCollection<NotebookUser>("Users");
+            this.users.Indexes.CreateOne(new CreateIndexModel<NotebookUser>(Builders<NotebookUser>.IndexKeys
+                .Ascending(nu => nu.TaskChainId)));
+        }
+
+        public static string MongoHost => Environment.GetEnvironmentVariable("MONGO_HOST") ?? "localhost";
+
+        public static string MongoPort => Environment.GetEnvironmentVariable("MONGO_PORT") ?? "localhost";

```

Comment:
port shouldn't be localhost

Reply:
It is not localhost, it is localhost if and only if the environment variable is not set, and the environment variable otherwise